frontend/apps/console/public/config.js

```
@@ -25,61 +25,57 @@ window.__THUNDERID_RUNTIME_CONFIG__ = {
         'getting-started/connect-your-application/prompts/node/redirect-based.txt',
       'applications.templates.node.llmPrompt.embedded':
         'getting-started/connect-your-application/prompts/node/embedded.txt',
       'applications.templates.browser.docs': 'getting-started/connect-your-application/browser/',
-      'applications.templates.browser.playground':
-        'https://stackblitz.com/fork/github/thunder-id/javascript-sdks/tree/main/samples/browser/quickstart?file=README.md',
+      'applications.templates.browser.playground': '',
       'applications.templates.browser.llmPrompt.redirectBased':
         'getting-started/connect-your-application/prompts/browser/redirect-based.txt',
       'applications.templates.browser.llmPrompt.embedded':
         'getting-started/connect-your-application/prompts/browser/embedded.txt',
       'applications.templates.android.docs': 'getting-started/connect-your-application/android/',
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)